packages/hooks/src/hooks.ts

```diff
@@ -68,3 +68,23 @@ export function useElementNode<ElementType = HTMLElement>() {
 
   return [element, elementRefCallback] as const;
 }
+
+export function useDocumentEventListener(
+  enabled: boolean,
+  type: string,
+  eventCallback: (e) => void,
```

Comment:
Nit: `e => void`

Reply:
Also, could we actually make the type for this an `EventCallback` or something?

Reply:
`EventListener` 👍